ElectronicObserverTypes/Serialization/FitBonus/FitBonusData.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Text.Json.Serialization;
+using System.Collections.Generic;
+
+namespace ElectronicObserverTypes.Serialization.FitBonus
```

Comment:
Use file scoped namespaces. I think if you just add a `;` at the end of this line, Visual Studio should do the rest.